src/components/Subscribe/index.js

```diff
@@ -79,23 +83,34 @@ const EmailSubscribe = () => {
   )
 }
 
-const Subscribe = ({ ...props }) => (
-  <div {...props}>
-    <div css={subscribeContainer}>
-      <p css={theme => [fontStyle(theme), subscribeTitle]}>
-        <FormattedMessage
-          id="subscribeToOurNewsLetter"
-          defaultMessage={`<span>SUBSCRIBE</span> to our newsletter`}
-          values={{
-            span: str => <span css={subscribe}>{str}</span>,
-          }}
-        />
-      </p>
-    </div>
-    <div css={emailAddressContainer}>
-      <EmailSubscribe />
+const Subscribe = ({ ...props }) => {
+  const isMobile = useMedia([true], false)
+  return (
+    <div {...props}>
+      <div css={subscribeContainer}>
+        <p css={theme => [fontStyle(theme), subscribeTitle]}>
+          <FormattedMessage
+            id="subscribeToOurNewsLetter"
+            defaultMessage={`<span>SUBSCRIBE</span> to our newsletter`}
+            values={{
+              span: str =>
+                isMobile ? (
+                  <span css={subscribe}>
+                    {str}
+                    <br />
+                  </span>
+                ) : (
+                  <span css={subscribe}>{str}</span>
```

Comment:
Instead of rendering different structure for mobile, add `display: block;` for mobile media-query in css `subscribe`. Remember: if styling can be done in css, use css to do it.

Reply:
I got it. This is cool and fixed.